hierarchy.py

```diff
@@ -0,0 +1,45 @@
+class BaseFile:
+    def __init__(self, filename, creator=''):
+        self.content = open(filename)
+        self.weight = self.content.__sizeof__()
+        self.creator = creator
+
+    def read(self, user_name: str):
+        if self.creator == user_name:
+            return self.content
+        return None
+
+
+class TextFile(BaseFile):
+    def count(self, string: str):
+        data = self.content.read()
+        return data.count(string)
```

Comment:
just narrow this to a single line, data is not a variable name that given any additional information